tsconfig.webpack.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "es6",
+    "target": "es2017"
```

Comment:
do we want to target es2017? Shouldn't this be es5? So we know that stuff works on all the browsers.

Reply:
Yeah, you're right. This should change back to es5, probably by just removing "target" from this file.